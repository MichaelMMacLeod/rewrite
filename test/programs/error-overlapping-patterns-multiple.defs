(def (a b $x ..) _)
(def (a b $x .. c) _)
(def (a b $x .. ($y ..) d) _)