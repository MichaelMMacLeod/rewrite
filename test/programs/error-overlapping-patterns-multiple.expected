error[E007]: overlapping patterns
./test/programs/error-overlapping-patterns-multiple.defs:1:6
   |
 1 | (def (a b $x ..) _)
   |      ^^^^^^^^^^^ this pattern may match the same term as ...
./test/programs/error-overlapping-patterns-multiple.defs:2:6
   |
 2 | (def (a b $x .. c) _)
   |      ^^^^^^^^^^^^^ ... this other pattern
help: patterns possibly matching the same term are not allowed

error[E007]: overlapping patterns
./test/programs/error-overlapping-patterns-multiple.defs:1:6
   |
 1 | (def (a b $x ..) _)
   |      ^^^^^^^^^^^ this pattern may match the same term as ...
./test/programs/error-overlapping-patterns-multiple.defs:3:6
   |
 3 | (def (a b $x .. ($y ..) d) _)
   |      ^^^^^^^^^^^^^^^^^^^^^ ... this other pattern
help: patterns possibly matching the same term are not allowed
